src/sql/workbench/contrib/executionPlan/browser/executionPlanPropertiesViewBase.ts

```diff
@@ -245,6 +272,68 @@ export abstract class ExecutionPlanPropertiesViewBase implements IVerticalSashLa
 		this._tableComponent.resizeCanvas();
 	}
 
+	private searchTable(searchString: string): void {
+		if (searchString === '') {
+			this._tableComponent.setData(this.flattenTableData(this._tableData, -1));
+		} else {
+			this._tableComponent.setData(
+				this.flattenTableData(
+					this.searchNestedTableData(searchString, this._tableData).data,
+					-1)
+			);
+		}
+		this._tableComponent.rerenderGrid();
+	}
+
+	private searchNestedTableData(pattern: string, data: Slick.SlickData[]): { include: boolean, data: Slick.SlickData[] } {
```

Comment:
Is pattern a regex for a data value or the target data value? If it's just the target data value than renaming it to something along those lines would make more sense.